appcelerator/titanium_mobile
# fix(android): Ti.Media.previewImage() fails to display in-memory blobs as of 9.1.0

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-28246

**Github Issue:**
https://github.com/appcelerator/titanium_mobile/issues/12265

**Summary:**
- Changed `TiFileHelper.getTempFile()` to create file like it did before 9.1.0.
- Fixed Java `TiFileHelper.getTempFileFromInputStream()` method which always failed to create file.
  * Prevented our `Ti.Media.previewImage()` method from working.
  * Also caused issues in 3rd party modules.
- Fixed issue where Java `TiFileHelper` would not delete files if app was force-quit.

**Test:**
1. Run the test shown in ticket [TIMOB-28246](https://jira.appcelerator.org/browse/TIMOB-28246).
2. Verify a screenshot of the app is now shown in preview window.
